cropper/src/main/java/com/canhub/cropper/CropImage.java

```diff
@@ -189,13 +189,7 @@ public static Intent getPickImageChooserIntent(
       allIntents.addAll(getCameraIntents(context, packageManager));
     }
 
-    List<Intent> galleryIntents =
-        getGalleryIntents(packageManager, Intent.ACTION_GET_CONTENT, includeDocuments);
-    if (galleryIntents.size() == 0) {
-      // if no intents found for get-content try pick intent action (Huawei P9).
```

Comment:
How did you fix the Huawei issue?

Huawei is to know having some problems with the internal path, mainly on Version N (API 24)

Reply:
Hmm, unfortunately I can't tell if it fixes the Huawei issue specifically since I don't have one. But the way it was getting that list of intents no longer works on API level 29+, so we have to change it. I think a Huawei fix would be a separate issue.